express-stock-trading-platform/server084.js

```diff
@@ -0,0 +1,39 @@
+const express = require('express');
+const app = express();
+
+app.listen(3000, function() {
+  console.log('Your app is listening on port 3000.');
+});
+
+app.get('/hi', (req, res) => {
+  res.send('Hi there trader!');
+});
+
+const prices = {
+  ABC: 48.83,
+  DEF: 2.98,
+  GHI: 3.99,
+  JKL: 99,
+  MNO: 45.38,
+  OPQ: 0.48,
+  RST: 9.32,
+  UVW: 10.94,
+  XYZ: 5.32
+};
+
+app.get('/buy/:ticker/:shares', (req, res) => {
+  const ticker = req.params.ticker;
+  const shares = req.params.shares;
+
+  const total = shares * prices[ticker];
+
+  res.send(
+    `Transaction complete, you purchased ${shares} shares of ${ticker} at $${
+      prices[ticker]
+    }/share for a total of $${total}.`
+  );
+});
+
+app.get('/sell/:ticker/:shares', (req, res) => {});
+
+// In the body of the callback function, create the `ticker`, `shares`, and `total` variables and assign them values similar to how they were created in the /buy route.
```

Comment:
// In the body of the callback function, create the `ticker`, `shares`, and `total` variables and assign them values similar to how they were created in the `/buy` route.